README.md

```diff
@@ -1,2 +1,127 @@
-# contributor-access-policy
+## Summary
+
 This document outlines the procedure for getting Contributor status to various Solana source code repositories and the Solana Tech Discord server.
+
+Everyone is welcome to contribute to the Solana codebase. Contributing doesn’t just mean submitting pull requests, 
+there are many different ways for you to get involved, including answering questions on Stack Exchange and Discord, reporting or triaging bugs.
+
+There are two types of Contributor status: 
+
+#### Contributor
+
+Those with the Contributor status have any level of access to the [Solana Program Library](https://github.com/solana-labs/solana-program-library) (SPL)
+repository and the Contributor role in the [Solana Tech Discord](https://discordapp.com/invite/pquxPsq).
```

Comment:
Yes that makes sense. Then maybe new Discord roles like:
* `SPL Triage`, `SPL Write`, `SPL Maintain`
*  `Labs Triage, `Labs Write`, `Labs Maintain`

In this setup the `Core Contributor` role on Discord seems obsolete

Reply:
do we care about github permissions on discord?  i'm just thinking about the hell that is managing discord permissions, then having to do it for seven roles instead of three

Reply:
Surfacing the current membership of all groups somehow is what I'm mainly interested in, along with being able to easily map users from github to discord so that when a discussion bridges the two platforms people can reasonably find each other.

Thank you github for this little gem btw:
<img width="755" alt="image" src="https://user-images.githubusercontent.com/1281082/213368589-847b9caf-15c0-4f61-83e2-fd8466c32934.png">

Maybe there's no need for linkage at all between discord/github?